.gitignore

```diff
@@ -44,4 +44,6 @@ e2e-test/image/e2e/manifests/
 *.o
 
 __pycache__
-*.pyc
\ No newline at end of file
+*.pyc
+
+docs/
```

Comment:
Why should we ignore the `docs`? (and it seems that the `docs` folder doesn't exist)

Reply:
When I run some build commands, this directory may be generated.

Reply:
Which build command?

Reply:
I am trying to reproduce it.

Reply:
I can't reproduce this issue. I will close this issue now.